esrally/mechanic/telemetry.py

```diff
@@ -774,14 +775,16 @@ class DiskIo(InternalTelemetryDevice):
     """
     Gathers disk I/O stats.
     """
-    def __init__(self, metrics_store, node_count_on_host):
+    def __init__(self, metrics_store, node_count_on_host, log_root, node_name):
```

Comment:
The `DockerLauncher` calls this constructor as well. Can you please update the constructor invocation accordingly and double-check you have adapted all call-sites?

Reply:
As per offline discussion, I fixed the constructor call. But uncovered that `attach_to_node` isn't called for telemetry devices in DockerLauncher and so DIskIo isn't working properly: opened https://github.com/elastic/rally/issues/733 to fix it separately.

Reply:
Thank you.